balance_service/enforcement/usage_enforcement.py

```diff
@@ -379,7 +410,7 @@ def _date_from_string(self, date_string, date_format="%Y-%m-%d %H:%M:%S"):
             # Blazar introduces a new dateformat
             new_date_format = "%Y-%m-%dT%H:%M:%S"
             date = datetime.datetime.strptime(date_string, new_date_format)
-        return date
+        return date.replace(tzinfo=pytz.UTC)
```

Comment:
Previously timezones were being ignored here. They didn't matter since they were just storing lease start and end times for charging, never doing any comparison. That isn't the case anymore because of `_check_lease_update_window` so we need to make them timezone aware.